src/datepicker/datepicker-i18n.ts

```diff
@@ -0,0 +1,11 @@
+import {Injectable} from '@angular/core';
+
+const WEEKDAYS = ['Su', 'Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa'];
+const MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+
+@Injectable()
+export class NgbDatepickerI18n {
```

Comment:
As with the calendar we need to split into an abstract class and a concrete implmentation(`NgbDatepickerI18nENLocale` or sth)